### PR TITLE
Add source: frankston_vic_gov_au

### PR DESCRIPTION
**Pull request by b-goold**

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/frankston_vic_gov_au.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import requests
+import json
+from shapely.geometry import Point, MultiPolygon, Polygon
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/frankston_vic_gov_au.py:3]: shapely currently is not a dependency. We tried adding shapely as dependency in the past and some people had problems running this integration at all (#1705)

Comment (b-goold) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/frankston_vic_gov_au.py:3]: Ah bugger. Thanks, I'll have a go at replicating the behaviour without Shapely.

Comment (b-goold) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/frankston_vic_gov_au.py]: Resolved with https://github.com/mampfes/hacs_waste_collection_schedule/pull/2141/commits/0205c5dfa4aea9aefbd481c73bfd9cc205a630f6

**Comment by 5ila5** [comment]
Thanks for your contribution.

Did some reformatting based on (.pre-commit-config.yaml)

**Pull request merged**